starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,30 @@
+body {
```

Comment:
in terms of CSS try to play around more with `padding` and `width` in order to make it more similar to the example :)